cou/steps/backup.py

```diff
@@ -44,9 +44,12 @@ async def backup(model: Model) -> Path:
     await model.run_on_unit(unit_name, f"chmod o+rx {basedir}")
 
     local_file = COU_DATA / os.path.basename(remote_file)
-    logger.info("SCP from  mysql-innodb-cluster:%s to %s ...", remote_file, local_file)
+    logger.info("SCP from mysql-innodb-cluster:%s to %s ...", remote_file, local_file)
     await model.scp_from_unit(unit_name, remote_file, str(local_file))
 
+    progress_indicator.info(f"Backed up MySQL to {local_file}")
+    progress_indicator.start()
```

Comment:
do you need to pass anything to .start here? Elsewhere, it's `.start(step.description)`.

Reply:
I do the local testing and it will use the previous text. So it should be fine I guess.